### PR TITLE
chore(docs): Update How to Oracles

### PR DESCRIPTION
**Pull request by critesjosh**
# Description

Updates the code examples in the how-to so that it works. I tested this with version 0.32, but I assume it didn't change from 0.31 to 0.32 so am updating that as well.

## Problem\*

Resolves https://github.com/AztecProtocol/dev-rel/issues/282
closes https://github.com/noir-lang/noir/issues/5407
## Documentation\*

Check one:
- [ ] No documentation needed.
- [x] Documentation included in this PR.
- [ ] **[For Experimental Features]** Documentation to be submitted in a separate PR.

# PR Checklist\*

- [x] I have tested the changes locally.
- [ ] I have formatted the changes with [Prettier](https://prettier.io/) and/or `cargo fmt` on default settings.

### CODE REVIEW

**Review by Savio-Sou — APPROVED** [review]
(Blind approval to unblock merging if desired)

**Comment by TomAFrench** [comment]
Can we avoid having large segments of code just in markdown? We added snippet functionality specifically in order to avoid potential for docs to become stale like this.

We can instead have these examples in code which is tested in CI and then pull snippets into the docs

**Comment by Savio-Sou** [comment]
> Can we avoid having large segments of code just in markdown?

Great if we could document how to make use of codegen: https://github.com/noir-lang/noir/issues/4940

**Comment by TomAFrench** [comment]
This doesn't rely on codegen but the snippet workflow.

Codegen is in the "forget" stage of "set and forget" anyhow.

**Comment by critesjosh** [comment]
Yes I can update it

File: docs/versioned_docs/version-v0.32.0/how_to/how-to-oracles.md [hunk 1]
```diff
@@ -137,38 +137,32 @@ app.listen(5555);
...
-interface ForeignCallResult {
-  values: ForeignCallParam[],
-}
+export type ForeignCallResult = {
+  values: (ForeignCallSingle | ForeignCallArray)[];
+};
 ```
 
-::: Multidimensional Arrays
+:::Multidimensional Arrays
```
Comment (critesjosh) [inline docs/versioned_docs/version-v0.32.0/how_to/how-to-oracles.md:163]:
```suggestion
:::info Multidimensional Arrays
```

**Comment by critesjosh** [comment]
@TomAFrench this example code requires running an json rpc server in js. do we have any examples of how to approach this? Should it be added in `examples/`?

**Comment by critesjosh** [comment]
i am going to merge the updates since they are working and the current published docs are not. we can revisit fully testing and pulling in snippets in another pr

**Pull request merged**